keystore.go

```diff
@@ -2,47 +2,48 @@ package main
 
 import (
 	"encoding/json"
-	"errors"
+	"fmt"
 	"io"
 	"io/ioutil"
 
 	"github.com/prysmaticlabs/eth1-mock-rpc/eth1"
-	"github.com/prysmaticlabs/prysm/shared/keystore"
```

Comment:
Would it make sense to implement this in prysm then import it here? Or is the plan to do the inverse?